pkg/ipmasq/ipmasq.go

```diff
@@ -33,6 +33,22 @@ import (
 
 var (
 	log = logging.DefaultLogger.WithField(logfields.LogSubsys, "ipmasq")
+
+	defaultNonMasqCIDRs = map[string]net.IPNet{
+		"10.0.0.0/8":      mustParseCIDR("10.0.0.0/8"),
+		"172.16.0.0/12":   mustParseCIDR("172.16.0.0/12"),
+		"192.168.0.0/16":  mustParseCIDR("192.168.0.0/16"),
+		"100.64.0.0/10":   mustParseCIDR("100.64.0.0/10"),
+		"192.0.0.0/24":    mustParseCIDR("192.0.0.0/24"),
+		"192.0.2.0/24":    mustParseCIDR("192.0.2.0/24"),
+		"192.88.99.0/24":  mustParseCIDR("192.88.99.0/24"),
+		"198.18.0.0/15":   mustParseCIDR("198.18.0.0/15"),
+		"198.51.100.0/24": mustParseCIDR("198.51.100.0/24"),
+		"203.0.113.0/24":  mustParseCIDR("203.0.113.0/24"),
+		"240.0.0.0/4":     mustParseCIDR("240.0.0.0/4"),
```

Comment:
it would be nice to have a comment where these CIDRs came from. Also some documentation updates WRT would be helpful for users.

Reply:
Added a comment about the CIDRs origin. The documentation will come in a separate PR.